marconi-cardano-core/src/Marconi/Cardano/Core/Types.hs

```diff
@@ -118,6 +117,9 @@ type TxOutRef = C.TxIn
 txOutRef :: C.TxId -> C.TxIx -> C.TxIn
 txOutRef = C.TxIn
 
+data IndexingDepth = MinIndexingDepth !Word64 | MaxIndexingDepth
+  deriving (Eq, Show, Generic, Aeson.FromJSON, Aeson.ToJSON)
+
 newtype ShouldFailIfResync = ShouldFailIfResync Bool
```

Comment:
I believe this definition of `ShouldFailIfResync` also should be deleted. But if you prefer, that could go in a small follow-up ticket since PLT-8635 doesn't mention `marconi-cardano-core`.

Reply:
This was another merge conflict incorrectly resolved. Also re-removed!